    let num = 1
    try {
        num.toUpperCase();
    }
    catch (error) {
        console.log('name', error.name);
        console.log('message', error.message);
    }



 
